cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/verification/FlowVerification.java

```diff
@@ -60,7 +61,7 @@
    * @return An instance of {@link VerifyResult} depending of status of verification.
    */
   @Override
-  public VerifyResult verify(Id.Application appId, final FlowSpecification input) {
+  public VerifyResult verify(ApplicationId appId, final FlowSpecification input) {
```

Comment:
changeing this class seems to unnecessarily add conversions between Id to EntityId. Unless you can change ``SimpleQueueSpecificationGenerator`` in this PR, better to revert this change.